cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -105,6 +128,8 @@
     private TlsSidecar tlsSidecar;
     private String tlsSidecarImage;
     private String minInsyncReplicas = "1";
+    private boolean authenticationEnabled;
+    private boolean authorizationEnabled;
```

Comment:
I am not sure about the naming of these ... I mean they should just reflect what they are in Cruise Control so "security" and "ssl". AFAIU from CC doc, the `webserver.security.enable` enables the HTTP basic authentication so why using the `isApiAuthorizationEnabled` method to fill the `authorizationEnabled` field? it's about enabling the HTTP basic authentication; for `webserver.ssl.enable`, it's enabling the SSL/TLS layer on HTTP, so HTTPS but it is not related to authentication, so why using `isApiAuthenticationEnabled` to fill the corresponding `authenticationEnabled`? I would say that fields should just be `securityEnabled` and `sslEnabled` and so the corresponding methods getting the config from CC. Am I misunderstading anything here?

Reply:
Or maybe you could use `authenticationEnabled` for the `webserver.security.enable` and something like `encryptionEnabled` for `webserver.ssl.enable`